### PR TITLE
upgrade zig codes to 0.11.0-dev.3379+629f0d23b

### PR DESCRIPTION
**Pull request by sjinzh**
If this PR is related to coding or code translation, please fill out the checklist and paste the console outputs to the PR.

- [x] I've tested the code and ensured the outputs are the same as the outputs of reference codes.
- [x] I've checked the codes (formatting, comments, indentation, file header, etc) carefully.
- [x] The code does not rely on a particular environment or IDE and can be executed on a standard system (Win, macOS, Ubuntu).

### CODE REVIEW

**Comment by sjinzh** [comment]
@krahets

File: codes/zig/chapter_stack_and_queue/linkedlist_deque.zig [hunk 1]
```diff
@@ -5,14 +5,14 @@
 const std = @import("std");
 const inc = @import("include");
 
-// 双向链表节点
+// 双向链表结点
```
Comment (krahets) [inline codes/zig/chapter_stack_and_queue/linkedlist_deque.zig:8]: We use "节点" for this book. Please replace them all.

**Review by krahets — COMMENTED** [review]
Thanks for the update! Please address the comment.

File: codes/zig/chapter_stack_and_queue/linkedlist_deque.zig (hunk 1 above)
Comment (sjinzh) [reply codes/zig/chapter_stack_and_queue/linkedlist_deque.zig]: > We use "节点" for this book. Please replace them all.

@krahets Done!

**Review by krahets — APPROVED** [review]
Thanks for the PR!

**Pull request merged**